src/it/scala/io/scalac/tezos/translator/CaptchaDirectiveSpec.scala

```diff
@@ -67,21 +72,43 @@ class CaptchaDirectiveSpec extends WordSpec with Matchers with ScalatestRouteTes
 
   "ReCaptcha directive" should {
 
-    "check captcha header" in {
+      "check captcha header" in {
+      val configWithScore= captchaTestConfig.copy(score=0.9F)
+        val captchaTestEndpoint: Endpoint[Option[Auth.Captcha], (DTO.ErrorDTO, StatusCode), String, Nothing] = Endpoints
+          .captchaEndpoint(configWithScore)
+          .in("test")
+          .get
+          .out(jsonBody[String])
+
+      val captchaTestRoute: Route = captchaTestEndpoint
+        .toRoute((ReCaptcha.withReCaptchaVerify(_, sy.log, captchaTestConfig)).andThenFirstE{ _: Unit => emptyOk() })
+
       Get(testEndpoint) ~> Route.seal(captchaTestRoute) ~> check {
         status shouldBe StatusCodes.BadRequest
         responseAs[Error].error shouldBe s"Request is missing required HTTP header '$headerName'"
       }
 
       val testResponse = "testResponse"
-      stubForCaptchaCheck(testResponse)
+      stubForCaptchaCheck(testResponse,1.0F)
```

Comment:
please add space after a comma

Reply:
ok, i didn't saw lack of space because of this intellij auto help
![image](https://user-images.githubusercontent.com/23175356/71714436-2f86ba80-2e0e-11ea-84b5-e9d325bcd140.png)